tests/integration/lxd/test_launcher.py

```diff
@@ -116,18 +116,14 @@ def get_instance_and_base_instance(get_base_instance, instance_name):
             base_instance.delete()
 
 
-# exclude 23.04 (lunar) because it is a daily, not a supported release
-@pytest.mark.parametrize(
```

Comment:
This test is about testing the launch command, not testing every single bases.

The integration tests in `tests/integration/lxd/test_remotes.py` is where we iterate through all the bases.